src/pyelliptic/eccblind.py

```diff
@@ -151,7 +150,7 @@ def _ec_point_serialize(self, point):
                 # padding manually
                 bx = OpenSSL.malloc(0, OpenSSL.BN_num_bytes(x))
                 OpenSSL.BN_bn2bin(x, bx)
-                out = bx.raw.rjust(l_, chr(0))
+                out = bx.raw.rjust(l_, chr(0).encode())
```

Comment:
@PeterSurda maybe use literal `b'\x00'` here?

Reply:
I haven't looked at this aspect of pyelliptic yet.

Reply:
it is rather an aspect of python3, where bytes and str are treated separately